Resources/public/js/views/universaldiscovery/ez-universaldiscoverysearchview.js

```diff
@@ -0,0 +1,500 @@
+
+            container.setHTML(this.template({
+                searchText: this.get('searchText'),
+                searchResultCount: this.get('searchResultCount'),
+                searchResultList: this._convertToJSONList(),
+                isFirst: this._isFirstPage(),
+                isLast: this._isLastPage(),
+                hasPages: this._hasPages(),
+                loadingError: this.get('loadingError'),
+                multiple: this.get('multiple')
+            }));
+
+            this.get('selectedView').set('contentStruct', null);
+            container.one('.ez-ud-search-selected').append(
+                this.get('selectedView').render().get('container')
+            );
+            this._uiPageEndLoading();
+            return this;
+        },
+
+        onUnselectContent: function (contentId) {
+            var selectedViewStruct = this.get('selectedView').get('contentStruct');
+
+            if ( selectedViewStruct && selectedViewStruct.contentInfo.get('id') === contentId ) {
+                this.get('selectedView').set('confirmButtonEnabled', true);
+            }
+        },
+
+        /**
+         * `multipleChange` and `isSelectableChange` events handler. It sets the selected view
+         * `addConfirmButton` flag according to the new `multiple` attribute value and passes
+         * new `isSelectable` function to the selected view.
+         *
+         * @method _setSelectedViewAttrs
+         * @protected
+         */
+        _setSelectedViewAttrs: function () {
+            this.get('selectedView').setAttrs({
+                'addConfirmButton': this.get('multiple'),
+                'isSelectable': this.get('isSelectable')
+            });
+        },
+
+        /**
+         * Search button `tap` event handler. It sets the attributes that take part in the search
+         * and proceeds with the firing location search.
+         *
+         * @method _search
+         * @protected
+         * @param {EventFacade} e
+         */
+        _search: function (e) {
+            var c = this.get('container'),
+                searchInput = c.one(e.target.getAttribute('data-search-input-selector')),
+                searchText = searchInput.get('value');
+
+            e.preventDefault();
+
+            this.set('searchText', searchText);
+            this.set('offset', 0);
+            this._fireLocationSearch();
+        },
+
+        /**
+         * Fires the `locationSearch` event to fetch the result list of the search.
+         *
+         * @method _fireLocationSearch
+         * @protected
+         */
+        _fireLocationSearch: function () {
+            var searchText = this.get('searchText');
+
+            this._uiPageLoading();
+
+            this.fire('locationSearch', {
+                viewName: 'udwsearch-' + searchText,
+                resultAttribute: 'searchResultList',
+                resultTotalCountAttribute: 'searchResultCount',
+                loadContentType: true,
```

Comment:
The UDW now has `loadContent` option as well which is passed to the methods https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/universaldiscovery/ez-universaldiscoverymethodbaseview.js#L107
so you should also set the `loadContent` flag on the event object.